src/developers/quick_start_guide.md

```diff
@@ -0,0 +1,55 @@
+This guide will take you through the steps of:
+- starting a `Notary` server
+- running a `Prover` to notarize some web data
+- running a `Verifier` to verify the notarized data
+
+Note that the TLSNotary protocol assumes that the `Notary` is trusted by the `Verifier`. To minimize the trust, the `Verifier` itself can act as a `Notary`.
+
+# Preliminaries
+
+### Install rust
+
```

Comment:
I was wondering why you were adding these extra arguments to curl, before I realised this comes from the rustup site.

Maybe add some extra text here:
```md
If you don't have `rust` installed yet, install it with [rustup](https://rustup.rs/):
```